libs/net/dal/Migrations/1.3.8/Down/PostDown/00-source-series.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+
+ALTER TABLE public."source" DROP COLUMN "is_cbra_source";
```

Comment:
Why do you need these?  EF migration should handle all of this for you.

Reply:
so I don't need this script?

Reply:
Not unless you need to do more than add a column

Reply:
updated. thanks!